test/e2e/libpod_suite_test.go

```diff
@@ -449,3 +449,14 @@ func (p *PodmanTest) GetContainerStatus() string {
 	session.WaitWithDefaultTimeout()
 	return session.OutputToString()
 }
+
+// BuildImage uses podman build and buildah to build an image
+// called imageName based on a string dockerfile
+func (p *PodmanTest) BuildImage(dockerfile, imageName string) {
```

Comment:
fingers crosses that this will work, but ipbabble's been having some fun getting Buildah to run in  a Docker container.

Reply:
works just fine